setup.py

```diff
@@ -4,15 +4,20 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
+import distutils.command.build
```

Comment:
I thought about that, but importing stuff into setup.py is a bit problematic, since it's not in a package and implicit relative imports don't work in Python 3...

Maybe I'll figure out if it's easily doable...